openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -586,6 +606,8 @@ class TaxonomyTagsView(ListAPIView, RetrieveUpdateDestroyAPIView):
           Using full_depth_threshold=1000 is recommended in general, but use lower values during development to ensure
           compatibility with both large and small result sizes.
         * include_counts (optional) - Include the count of how many times each tag has been used.
+        * include_perms (optional) - Include permission checks in response.
+          If omitted, permission fields (can_add_tag, can_change_tag, can_delete_tag) will be None.
```

Comment:
@bradenmacdonald I could do that by changing the Serializer depending on whether `include_perms` is present.

But since we've demonstrated that adding these permissions doesn't add any extra queries, I'm more inclined to remove the `include_perms` parameter and always include permissions in the response.

What would you prefer?

Reply:
Please see [786ef8b](https://github.com/openedx/openedx-learning/pull/138/commits/786ef8b137ddc93d5b6d6871316f70b56343db5d)